### DISCUSSION TITLE
Unable to create a Registry Key

### DISCUSSION THREAD

**Discussion by computerquip-work** [wixtoolset/issues#8544] [Questions]
It's been awhile since I've messed with WiX so this is probably a basic question. I wanted an installer that didn't do much other than create some registry keys. I had the following:

```
<Wix xmlns="http://wixtoolset.org/schemas/v4/wxs">
  <Fragment>
    <ComponentGroup Id="MAIN.COMPONENTS">
      <!-- Is a component needed here...? Is an explicit ID and/or GUID required...? -->
      <Component Id="RegistryEntries" Guid="{749C3926-1AFB-4F9B-9225-DC016659FCDB}">
        <RegistryKey
          Root="HKLM"
          Key="SYSTEM\bob"
          ForceCreateOnInstall="yes"
          ForceDeleteOnUninstall="yes"
        >
        </RegistryKey>
      </Component>
    </ComponentGroup>
  </Fragment>
</Wix>

```
This is giving me:
```
1>wix.exe : error WIX0204: ICE03: _Validation table: Data in KeyTAble column is incorrect; Table: FeatureComponents, Column: Component_, Key(s): WixDefaultFeature.RegistryEntries
1>obj\x64\Debug\en-US\<snip>.msi : warning WIX1076: ICE71: The Media table has no entries.
1>Done building project "<snip>.wixproj" -- FAILED.
```
I was trying to use the automatic feature table generation. If I specify an explicit feature table, the build succeeds (with the warning about no entries in the media table because there are none).
The explicit feature specification is simple:
```
    <Feature Id="Main">
      <ComponentGroupRef Id="MAIN.COMPONENTS" />
    </Feature>
```
I can't tell if I'm doing something wrong here or not. Some guidance would be appreciated.

EDIT: This is using v5.0.0 stable.

**Comment by barnson** [comment]
Sounds like a bug. Please open an issue with a repro repo.
